Microsoft.Diagnostics.Runtime.sln

```diff
@@ -9,6 +9,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Diagnostics.Runti
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Diagnostics.Runtime.UnitTests", "src\Microsoft.Diagnostics.Runtime.UnitTests\Microsoft.Diagnostics.Runtime.UnitTests.csproj", "{214A7EC8-4D32-4202-8D8B-1761743EE527}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Diagnostics.Runtime.Util", "src\Microsoft.Diagnostics.Util\Microsoft.Diagnostics.Runtime.Util.csproj", "{BFC539C3-6E60-42DF-B68E-23315B337CD2}"
```

Comment:
```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Diagnostics.Runtime.Utilities", "src\Microsoft.Diagnostics.Utilities\Microsoft.Diagnostics.Runtime.Utilities.csproj", "{BFC539C3-6E60-42DF-B68E-23315B337CD2}"
```

💡 Consider avoiding abbreviations.

Is it by design that the directory name and project name are different?

Reply:
It takes me about 3x as long to figure out what I want to name something than it does write the code.  The utilities library now is just a place to park code, I am planning to actually name it and organize it in the next couple of days.  I'll likely name the library (and .csproj) `Microsoft.Diagnostics.Utilities` and the debugger wrappers `[assemblyname].DbgEng` since that's the name of the debugger.